tests/test_lizzy.py

```diff
@@ -180,10 +180,14 @@ def test_new_stack(monkeypatch,
             'parameters': parameters,
             'disable_rollback': disable_rollback,
             'dry_run': dry_run,
-            'region': region,
             'senza_yaml': "{MyDefinition: Values}\n",
             'stack_version': version,
             'tags': tags}
+
+    # only pass region when it is not None
+    if region:
+        data['region'] = region
```

Comment:
That is another reason why I do not like this `pytest.mark.parametrize` kind of test.